template.env

```diff
@@ -2,6 +2,15 @@ MISP_TAG=v2.4.175
 MODULES_TAG=v2.4.175
 PHP_VER=20190902
 
+PYPI_REDIS_VERSION="==5.0.*"
+PYPI_LIEF_VERSION=">=0.13.1"
+PYPI_PYDEEP2_VERSION="==0.5.*"
+PYPI_PYTHON_MAGIC_VERSION="==0.4.*"
+PYPI_MISP_LIB_STIX2_VERSION="==3.0.*"
```

Comment:
Name of the vars should match name of the package

Reply:
You want to drop the PYPI_ prefix?

Reply:
Nope, sorry I meant 'MISP_LIB_STIX2' for example.
Is it called like that? If so then ignore my previous comment